migrate/labels.yml

```diff
@@ -0,0 +1,122 @@
+### MORE THAN ONE OF THESE MAY APPLY AT A TIME.
+
+- name: "maintenance"
+  color: "ff9125"  # construction orange
+  description: "Relates to platform health and/or routine upkeep"
+
+- name: "security"
+  color: "f0d165"  # caution yellow
+  description: "Relates to improving to the security posture of the platform"
+
+- name: "enhancement"
+  color: "a2eeef"  # electric blue!
+  description: "Relates to new features or improvements to existing features"
+
+- name: "documentation"
+  color: "0052cc"  # royal blue
+  description: "Relates to documentation improvements"
+
+
+### LABELS INDICATING THE SCOPE OR FUNCTION OF THE ISSUE.
+### AT MOST ONE OF THESE SHOULD BE USED AT A TIME.
+
+- name: "depr"
+  color: "3c3cc3"  # indigo-blue
+  description: "Proposal for deprecation & removal per OEP-21"
+
+- name: "bug"
+  color: "d93f0b"  # scarlet red...
+  description: "Report of or fix for something that isn't working as intended"
+
+- name: "discovery"
+  color: "d876e3"  # a curious shade of lavendar
+  description: "Pre-work to determine if an idea is feasible"
+
+- name: "epic"
+  color: "7e027e"  # epic violet
+  description: "Large unit of work, consisting of multiple tasks"
+
+- name: "initiative"
+  color: "340963"  # formidably deep mauve
+  description: "Huge unit of work, consisting of multiple epics"
+
+
+### LABELS INDICATING ISSUE OUTCOMES.
+
+- name: "wontfix"
+  color: "ffffff"  # white
+  description: "This will not be worked on"
+
+- name: "duplicate"
+  color: "cfd3d7"  # duplicitous grey
+  description: "This issue or pull request already exists elsewhere"
+
+
+### LABELS USED IN THE OPEN EDX CONTRIBUTION WORKFLOW.
+### PLEASE CONSULT WITH OPEN EDX PROJECT MANAGMENT BEFORE CHANGING THESE.
+
+- name: "waiting on author"
+  color: "bfd6f6"  # baby blue
+  description: "PR reviewers are waiting on the author to answer questions, fix tests, etc."
+
+- name: "inactive"
+  color: "cc950a"  # clay-ish brown
+  description: "PR author has been unresponsive for several months"
+
+- name: "closed inactivity"
+  color: "dbcd00"  # gold
+  description: "PR was closed because the author abandoned it"
+
+- name: "needs test run"
+  color: "f5424b"  # crimson red
+  description: "Author's first PR to a repository, awaiting test authorization from Axim"
+
+- name: "core contributor"
+  color: "000000"  # black
+  description: "PR is Core Contributor with rights in this repository (manually added)"
+
+- name: "open source contribution"
+  color: "f0f0f0"  # light grey
+  description: "PR author is not from Axim or 2U"
+
+- name: "blocked by other work"
+  color: "ffa500"  # yellow orange
+  description: "PR cannot be finished until other work is complete"
+
+- name: "changes requested"
+  color: "76a4e3"  # cornflower blue
+  description: "PR reviewers are waiting on author to apply feedback"
+
+- name: "product review"
+  color: "c97bf7"  # light puple
+  description: "PR requires product review before merging"
```

Comment:
@nedbat Would you rather have `blended` be added here, or would you rather continue letting the OSPR bot do that?

Reply:
@nedbat For the time being, the bot will still apply a few of labels to PRs (just `blended` and `open-source-contribution` if I read [labels.py](https://github.com/openedx/openedx-webhooks/blob/master/openedx_webhooks/labels.py) correctly), even though it won't sync all of [labels.yml](https://github.com/openedx/openedx-webhooks-data/blob/main/labels.yaml) any more. I'm not sure if we actually need to define `blended` here in order for that to work, but I'll do it anyway just for completeness.